homeassistant/components/broadlink/config_flow.py

```diff
@@ -5,11 +5,7 @@
 import socket
 
 import broadlink as blk
-from broadlink.exceptions import (
-    AuthenticationError,
-    BroadlinkException,
-    NetworkTimeoutError,
-)
+from broadlink import exceptions as e
```

Comment:
May I suggest a different namespace? `e` seems very generic
```suggestion
from broadlink import exceptions as blk_e
```

Reply:
Nope, it's ugly :laughing:

It's not generic when we're inside Broadlink integration. We can add the prefix if at some point we need to import exceptions from other modules, which I don't think will happen, the plan is to stick to this one.

Reply:
I agree with @epenet this is worse than it was IMHO.

Reply:
Should we change cv to hass_cv? dt to hass_dt? This would only make reading the code less pleasant.

Namespaces look best when they're short. Of course, we won't go too far in using all the letters of the alphabet in the same file, but importing the exception module as "e" is quite common.

Reply:
I'd just stick to what we have already. Alternatively write out full words, but that may be too long for some tastes.